romana-install/roles/romana/install-master/files/build_romana_core

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+export GOPATH=/var/tmp/gopath
+export PATH=/usr/local/go/bin:$PATH
+
+go get -d github.com/romana/core/...
```

Comment:
this is cool feature! amazing.
any chance we can support branches ? probably something like
{% if romana_core_branch is defined %}cd $GOPATH/src/github.com/romana/core/;git checkout {{ romana_core_branch }}{% endif %}